sites/test-site/tailwind.config.js

```diff
@@ -0,0 +1,10 @@
+const { getPackageTailwindConfig, mergeConfigs } = require(
+  '@bodiless/cli'
+);
+// We need the line below only for watching for changes.
+const siteConfig = require('./site.tailwind.config');
```

Comment:
Do we still need a sepqrate site.tailwind.config here?  Can we not just add getPackageTailwindConfig to the site's tailwind.config.js?

Reply:
No, we don't need it to gather configs. But requiring it directly allows updating styles instantly during development.
I've commented it out and left a comment for site builders.